CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [1.0.5]
```

Comment:
Did 1.0.4 miss the changelog or something?

Reply:
You'll have to ask @jhuff078 about what happened there.